CHANGES.rst

```diff
@@ -253,6 +253,9 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Properly handle opening of FITS files from URL Requests and HTTPResponses
+  from remote URLs. [#6378]
```

Comment:
*Requests* and *HTTPResponses* with the plural form may be confusing. If we care only about Python 3, you could use instead the complete name (`http.client.HTTPResponse`) or just mention it as the *output of `urlopen`* ?